roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedImageView.java

```diff
@@ -337,12 +387,42 @@ private void updateAttrs(Drawable drawable) {
     }
   }
 
+  private void updateBackgroundAttrs(Drawable drawable) {
+    if (drawable == null) { return; }
+
+    if (drawable instanceof RoundedDrawable) {
+      ((RoundedDrawable) drawable)
+              //It's expected for a Background to fit the drawable to
+              //cover all background, so its ScaleType should be FIT_XY
+              .setScaleType(ScaleType.FIT_XY)
```

Comment:
If this is the only difference, let's just change the signature of `updateAttrs(Drawable drawable)` to `updateAttrs(Drawable drawable, ScaleType)` or `updateAttrs(Drawable, boolean background)`